scripts/test_images.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+# Consider command as failed when any component of the pipe fails:
+# https://stackoverflow.com/questions/1221833/pipe-output-and-capture-exit-status-in-bash
+set -o pipefail
+
+source ./scripts/test_lib.sh
+source ./scripts/build_lib.sh
+
+function startContainer {
+    # run docker in the background
+    docker run -d --rm --name "${RUN_NAME}" "${TAG}" &
+
+    # wait for etcd daemon to bootstrap
+    sleep 5
+}
+
+function stopContainer {
+    docker stop "${RUN_NAME}"
+    docker image rm "${TAG}"
+}
+
+# Can't proceed without docker
+if ! command -v docker >/dev/null; then
+    log_error "cannot find docker"
+    exit 1
+fi
+
+# You can't run darwin binaries in linux containers
+if [[ $(go env GOOS) == "darwin" ]]; then
+    echo "Please use linux machine for release builds."
+    exit 1
+fi
+
+# Docker build
+ARCH=$(go env GOARCH)
+DOCKERFILE="Dockerfile-release.${ARCH}"
+BINARYDIR=${BINARYDIR:-"bin"}
+# Pick defaults based on release workflow
+IMAGE=${IMAGE:-"gcr.io/etcd-development/etcd"}
+VERSION=${VERSION:-"3.6.99-amd64"}
+TAG="${IMAGE}:${VERSION}"
```

Comment:
User must explicitly provide a version, otherwise fail the test. 

Suggest to follow the name convention. 

```suggestion
REPOSITORY=${IMAGE:-"gcr.io/etcd-development/etcd"}
TAG=${VERSION}-${ARCH}
IMAGE=${REPOSITORY}:${TAG}
```
```